fontforge/ufo.c

```diff
@@ -113,9 +113,10 @@ static void extractNumericVersion(const char * textVersion, int * versionMajor,
 
 static void injectNumericVersion(char ** textVersion, int versionMajor, int versionMinor) {
   // We generate a version string from numeric values if available.
-  if (versionMajor == -1) asprintf(textVersion, "%s", "");
-  else if (versionMinor == -1) asprintf(textVersion, "%d", versionMajor);
-  else asprintf(textVersion, "%d.%d", versionMajor, versionMinor);
+  int err = 0;
```

Comment:
I don't really like this kind of error handling, it's basically `try...catch` -> do nothing; I'd rather just have the compiler warn about it if we're not doing anything with it than to add code to swallow it.

Reply:
This happened because there is so much warning garbage in ufo.c that I missed something important. This does indeed make the code less readable, but it makes the warnings a lot more useful.